mobile/lib/widgets/common/app_bar_dialog/app_bar_server_info.dart

```diff
@@ -45,17 +78,45 @@ class AppBarServerInfo extends HookConsumerWidget {
           child: Column(
             crossAxisAlignment: CrossAxisAlignment.center,
             children: [
-              Padding(
-                padding: const EdgeInsets.all(8.0),
-                child: Text(
-                  serverInfoState.isVersionMismatch
-                      ? serverInfoState.versionMismatchErrorMessage
-                      : "profile_drawer_client_server_up_to_date".tr(),
-                  textAlign: TextAlign.center,
-                  style: TextStyle(fontSize: 11, color: context.primaryColor, fontWeight: FontWeight.w500),
+              if (showWarning) ...[
+                SizedBox(
+                  width: double.infinity,
+                  child: Container(
+                    decoration: const BoxDecoration(
+                      color: Color.fromARGB(80, 243, 188, 106),
+                      borderRadius: BorderRadius.all(Radius.circular(8)),
+                    ),
+                    padding: const EdgeInsets.symmetric(horizontal: 12, vertical: 6),
+                    child: Row(
+                      mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                      children: [
+                        Expanded(
+                          child: Text(
+                            serverInfoState.versionMismatchErrorMessage,
+                            textAlign: (serverInfoState.isClientOutOfDate || serverInfoState.isServerOutOfDate)
+                                ? TextAlign.start
+                                : TextAlign.center,
+                            style: const TextStyle(fontSize: 13, fontWeight: FontWeight.w500),
+                            maxLines: 3,
+                            overflow: TextOverflow.ellipsis,
+                          ),
+                        ),
+                        if (serverInfoState.isClientOutOfDate || serverInfoState.isServerOutOfDate)
```

Comment:
So the conditions are already in `showWarning`, can you make a local variable to make the condition easier to understand for this specific case?